Added List Users, Query User by ID, Update and Delete User

5r1n1 opened the pull request:
First commit and PR. Group operations, Bulk operations and expected behaviour for failure scenarios to be tested

5r1n1 committed:
Added List Users, Query User by ID, Update and Delete User

vaswadhati commented:
Please `npm run lint`, auto/fix and push again. Thanks!


>  101:7   error  Unexpected space between function name and paren  func-call-spacing
>  102:63  error  Missing semicolon                                 semi
>  105:53  error  Block must not be padded by blank lines           padded-blocks
>  115:56  error  Missing semicolon                                 semi
>  116:45  error  Missing semicolon                                 semi
>  121:58  error  Block must not be padded by blank lines           padded-blocks
>  125:49  error  Missing semicolon                                 semi
>  129:56  error  Missing semicolon                                 semi
>  130:45  error  Missing semicolon                                 semi
>  132:1   error  Trailing spaces not allowed                       no-trailing-spaces
>  133:53  error  Block must not be padded by blank lines           padded-blocks
>  145:75  error  Missing semicolon                                 semi
>  146:45  error  Missing semicolon                                 semi
>  149:53  error  Block must not be padded by blank lines           padded-blocks
>  153:49  error  Missing semicolon                                 semi

5r1n1 committed:
Running Lint first before testing now :)

1. More SCIM tests have been added.
2. Known bug: Groups endpoint is not listing groups (though totalRecords response is ok!)

vaswadhati commented:
No Lint Error. Yay. Thanks!

NOTE: I did not squash merge, so you can see the history of your commits. Going forward, when there are a number of commits to a logical chunk of work, I would squash and merge.

Closing
